documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -94,6 +94,11 @@ Capacity limits can be set for the following broker resources, using the standar
 
 Because Strimzi Kafka brokers are homogeneous, Cruise Control applies the same capacity limits to every broker it is monitoring.
 
+[NOTE]
+====
+Cruise Control assumes brokers resources to be homogeneous. Therefore, it is recommended to set value in `Kafka.spec.kafka.resources.requests` equal to `Kafka.spec.kafka.resources.limits`, so that Kubernetes will ensure broker pods have the same capacity and Guaranteed QoS, which also improves stability. This is especially important when using nodes with heterogeneous resources (CPU cores for example).
```

Comment:
"Cruise Control" -> "Strimzi Cruise Control"

It is recommended to set `Kafka.spec.kafka.resources.limits.cpu` so Kubernetes will ensure broker pods have the same CPU capacity, otherwise the CPU utilization for brokers can get under or overestimated when working with nodes with heterogeneous CPU resources. 

I need to think more about the setting of `Kafka.spec.kafka.resources.limits.cpu`. From what I understand, Cruise Control doesn't have any explicit protections against heterogeneous memory capacities but memory is considered when calculating broker load by using certain metrics (e.g. by using metrics like bytes in and bytes out in its CPU and Network utilization calculations). Values like bytes in and bytes out could differ between brokers with different memory capacities. I am not sure how big of an impact it would have or if it is covered by the other metrics but I would think it would be a good idea to set `Kafka.spec.kafka.resources.limits.cpu` (except for the caveats that Jakub mentioned!)